job_definitions/stats_snapshots.yml

```diff
@@ -38,3 +51,58 @@
               scripts/snapshot-framework-stats.py "{{ framework }}" "{{ environment }}"
 {% endfor %}
 {% endfor %}
+
+{% for environment in environments %}
+{% for framework, drive_folder, enabled in frameworks %}
+- job:
+    name: "export-framework-applicant-details-{{ environment }}"
+    display-name: "Export framework applicant details - {{ environment }}"
```

Comment:
Let's change these bits to `Daily framework application statistics export` or similar?